include/procfile.bash

```diff
@@ -18,7 +18,7 @@ procfile-exec() {
 	procfile-load-env
 	procfile-load-profile
 	cd "$app_path"
-	unprivileged /bin/bash -c "$(eval echo $@)"
+	exec setuidgid "$unprivileged_user" $(eval echo $@)
```

Comment:
reverting this change does not pass signals though. and adding exec to unprivileged yielded a broken build due to missing buildpack environment artifacts (i.e. `/app/.profile.d/nodejs.sh`)

so i am unclear as to what you are recommending here.